### PR TITLE
Code restructure

### PR DESCRIPTION
**Pull request by grace227**
Separating layout and callback functions based on the component module.

### CODE REVIEW

File: client/src/bl531_app.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+
+def get_beamline_components(BL_API_URL, BL_API_KEY, BL_UID):
+    # Get beamline PVs from MongoDB as OphydDash object
+    url = f"{BL_API_URL}/beamline/{BL_UID}/components"
+    print(url)
+    response = requests.get(url, headers={"api_key": BL_API_KEY})
+    print(response)
+    if response.status_code != 200:
+        raise BeamlineComponentsNotFound(f"Status code: {response.status_code}")
```
Comment (taxe10) [inline client/src/bl531_app.py:22]: Let's also import the BeamlineComponentsNotFound class in this script

File: client/src/bl531_app.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        external_stylesheets = [
+            dbc.themes.BOOTSTRAP,
+            "../assets/style.css",
+            "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css",
+        ]
+        self.app = Dash(__name__, external_stylesheets=external_stylesheets)
+        self.app.title = title
+        self.app._favicon = favicon
+
+    # def run(self, port="8053", host="0,0,0,0"):
```
Comment (taxe10) [inline client/src/bl531_app.py:101]: We should also clean up some comments in the script that are no longer needed

File: client/src/layout/app_layout.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                    dcc.Store(id="livescaler-cache", data=None),
+                ],
+                fluid=True,
+            ),
+        ]
+    )
+    return layout
+
+
+# def header(src_app_logo, logo_height, app_title):
```
Comment (taxe10) [inline client/src/layout/app_layout.py:44]: Same comment here

**Review by taxe10 — CHANGES_REQUESTED** [review]
All looks good. I left some minor comments that should be addressed before merging.

File: client/src/bl531_app.py (hunk 1 above)
Comment (grace227) [reply client/src/bl531_app.py:19]: Created the `BeamlineComponentsNotFound` in `client/src/model.py' and it's imported to `bl531_app.py`

**Review by taxe10 — APPROVED** [review]

**Pull request merged**